src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/ManageCaseDocumentsController.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.controllers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.UploadCaseFilesAboutToSubmitHandler;
+
+import static uk.gov.hmcts.reform.finrem.caseorchestration.OrchestrationConstants.AUTHORIZATION_HEADER;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(value = "/case-orchestration")
+public class ManageCaseDocumentsController extends BaseController {
```

Comment:
Please migrate this controller to a handler as per dfr1018

Reply:
Hi Mike, thanks yah currently working on it to migrate to the generic controller as per DFR-1018 but i had a chat with Ruban and we are of the opinion the PR could be reviewed while we work on migrating it to the new way of controller work